src/brevitas_examples/common/learned_round/learned_round_method.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from abc import ABC
+from abc import abstractmethod
+from typing import Callable, Dict, Generator, List, Optional, Tuple, Type
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from brevitas.core.function_wrapper.learned_round import LearnedRoundSte
+from brevitas.inject.enum import FloatToIntImplType
+from brevitas.inject.enum import LearnedRoundImplType
+from brevitas.nn.quant_layer import QuantWeightBiasInputOutputLayer as QuantWBIOL
+
+
+class StopFwdException(Exception):
```

Comment:
I'm sure we define this somewhere else, maybe GPTQ. Let's merge into one and put it in torch_utils.py